rspec-mode.el

```diff
@@ -141,12 +143,22 @@
   :type 'boolean
   :group 'rspec-mode)
 
+(defcustom rspec-vagrant-cwd "~/"
```

Comment:
I'm not sure. Looking at the docs there's `config.vm.synced_folder` which could be a way to determine that, but that can point to more than one folder, and I think it would be hard to guess. Either way, I think a more appropriate default value for this variable would be `/vagrant` which is the default value for `config.vm.synced_folder`.